core/src/mindustry/ui/dialogs/PlanetDialog.java

```diff
@@ -1160,7 +1160,7 @@ void playSelected(){
                 //free launch.
                 control.playSector(sector);
             }else{
-                CoreBlock block = from.planet.allowLaunchSchematics ? (from.info.bestCoreType instanceof CoreBlock b ? b : (CoreBlock)from.planet.defaultCore) : (CoreBlock)from.planet.defaultCore;
+                CoreBlock block = sector.allowLaunchSchematics() ? (sector.info.bestCoreType instanceof CoreBlock b ? b : (CoreBlock)sector.planet.defaultCore) : (CoreBlock)sector.planet.defaultCore;
```

Comment:
Why are you changing sector.info.bestCoreType? That looks very dangerous. You should care about the best core from the **launch** sector.